frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -56,6 +56,29 @@ export const getSurveyEventName = (surveyName: string): string => {
     return `${surveyName} survey sent`
 }
 
+const SURVEY_ANSWER_PROPERTY = '$survey_answer'
```

Comment:
bit confused what's this for, the site app & docs say the response prop is `$survey_response` ?